packages/react-reconciler/src/ReactFiberCompleteWork.new.js

```diff
@@ -1550,8 +1550,7 @@ function completeWork(
           // Run passive effects to retain/release the cache.
           workInProgress.flags |= Passive;
         }
-        const spawnedCachePool: SpawnedCachePool | null = (workInProgress.updateQueue: any);
-        if (spawnedCachePool !== null) {
+        if (workInProgress.alternate !== null) {
```

Comment:
Nit: Can you replace all the `workInProgress.alternate` occurrences in this block to `current` instead? Slightly more optimal because there are fewer property lookups, and also fewer places we need to update when `.alternate` is renamed.